app/style.css

```
@@ -1,7 +1,237 @@
 @import "tailwindcss";
 
+@variant dark (&:is(.dark *));
```

コード内にテーマの出典情報があると良いと思います。

```suggestion
/*
Intent UI Themes
https://intentui.com/themes

- Gray Color: Neutral
- Accent Color: Neutral
- Primary Color: Orange
- Radius: 0.5
*/
@variant dark (&:is(.dark *));
```